Automattic/simplenote-android
# App Bar Title

### Fix
Remove the `TypefaceSpan` from the app bar title in order to use the default title font for all screens.  See the screenshots below for illustration.

![app_bar_title](https://user-images.githubusercontent.com/3827611/65975124-64f4c400-e42b-11e9-8738-b073c5730137.png)

### Test
1. Notice ***All Notes*** title text weight.
2. Open navigation drawer.
3. Tap ***Settings*** item in drawer.
4. Notice ***Settings*** title text weight.
5. Tap back arrow in app bar.
6. Open navigation drawer.
7. Tap ***Edit*** button in ***Tags*** section.
8. Notice ***Edit tags*** title text weight.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
